core/modules/node/tests/node.test

```diff
@@ -950,6 +1046,31 @@ class NodeLayoutRevisionTestCase extends BackdropWebTestCase {
       // was set properly.
       $this->assertText($body, "\"{$body}\" found.");
     }
+
+    // Enable the node revision override layout 'custom_node_revision' with
+    // path node/%/revisions/%/view.
+
+    $layouts = layout_load_all();
+    $preview_layout = &$layouts['custom_node_revision'];
```

Comment:
This `&` is not needed...

Reply:
Agreed. (Also renamed the variable: should be `$revision_layout`.)